metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/dao/WikidataAccessDao.java

```diff
@@ -64,7 +77,7 @@ public EntityConverterUtils getEntityConverterUtils() {
   public final void init() throws WikidataAccessException {
```

Comment:
Adjust java doc: there is no more file parameter. The same goes for the parameter outputFile in the function getEntity below.

Reply:
done